README.md

```diff
@@ -1,2 +1,54 @@
 # resource-auditing
-Audit changes to K8s resources (e.g. NetworkPolicies)
+
+## Overview
+Project resource-auditing provides an auditing system for network policy-related, Kubernetes and Antrea custom-defined resources that tracks creations, reconfigurations, and deletions of these resources, stored as YAML files backed by a Git repository. The system also comes with a CLI for querying and filtering the repository for changes based on filters like date range or service account responsible for those changes, as well as a tagging and rollback feature for reverting the cluster state if the current cluster state is undesirable. A webUI service is linked to the repository, allowing for easy visualization of the entire history of resource configurations.
+
+## Getting Started
+ * ensure `kubectl` is running correctly prior to getting started
+
+A label is used to specify the node to run all audit services on, and it can be applied with:
+```bash
+kubectl label nodes <node-name> audit=target
+```
+Connect to the control node and copy `audit-policy.yaml` and `audit-config.yaml` to `/etc/kubernetes/manifests`. Run the following commands to create the directory the repository will be stored on.
+```bash
+sudo mkdir -m777 /data
+sudo mkdir -m777 /data/antrea-audit
```

Comment:
I would think that
```
mkdir -p /data/antrea-audit
```
is sufficient. let the user figure out if he needs to run as sudo. And not sure what the permissions are about, but 777 doesn't seem right

Reply:
777 is to allow read-write permissions for those other than the user creating it.